fontc_crater/src/args.rs

```diff
@@ -26,21 +26,22 @@ pub(super) struct RunArgs {
     /// Reusing this directory saves us having to clone all the repos on each run.
     ///
     /// This directory is also used to write cached results during repo discovery.
+    #[arg(default_value = "./font_repos")]
     pub(super) font_cache: PathBuf,
     /// Path to local checkout of google/fonts repository
-    #[arg(short, long)]
+    #[arg(short, long, default_value = "../fonts")]
     pub(super) fonts_repo: Option<PathBuf>,
     /// Optional path to write out results (as json)
-    #[arg(short = 'o', long = "out")]
+    #[arg(short = 'o', long = "out", default_value = "./font_repos/results.json")]
```

Comment:
- If I've set a different cache dir above, this will still default to here? 
- The current default is to just print a summary to stdout, which is useful sometimes
- if we're deciding that this is always set than there are various places where we're checking if `out_path.is_some()` or similar, and changing our behaviour based on that...

overall my instinct would be to just leave this part, in my actual use of this tool (outside of CI) I am generally comparing multiple runs and so I almost always want the results to have descriptive names, and if a new run overwrote my last result I would be annoyed.

Reply:
I think I can live without this one. Having font_repos and the location of google fonts default is very helpful because it makes a copy/pasteable command just about plausible instead of having to copy, fiddle with, and run.

Reply:
(removed)